com.unity.render-pipelines.core/Tests/Editor/RemoveRange.Extensions.Tests.cs

```diff
@@ -0,0 +1,164 @@
+using System;
+using System.Collections;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine.Rendering;
+
+namespace UnityEditor.Rendering.Tests
+{
+    public class RemoveRangeExtensionsTests
+    {
+        static TestCaseData[] s_ListTestsCaseDatas =
+        {
+            new TestCaseData(new int[] {1,2,3,4,5,6}, 1, 2).SetName("Remove middle"),
+            new TestCaseData(new int[] {1,2,3,4,5,6}, 0, 2).SetName("Remove front"),
+            new TestCaseData(new int[] {1,2,3,4,5,6}, 0, 6).SetName("Remove all"),
+            new TestCaseData(new int[] {1,2,3,4,5,6}, 5, 1).SetName("Remove back"),
+            new TestCaseData(new int[] {1,2,3,4,5,6}, 0, 0).SetName("Index 0"),
```

Comment:
Index 0 should work. But you should test negative index if you try to catch exception no?